src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerRequestTranscriptionIntTest.java

```diff
@@ -0,0 +1,364 @@
+
+        dartsDatabase.saveAll(hearing);
+    }
+
+    @Test
+    @Order(1)
+    @Transactional
+    void transcriptionRequestWithValidValuesShouldReturnSuccess() throws Exception {
+        TranscriptionUrgencyEnum transcriptionUrgencyEnum = TranscriptionUrgencyEnum.STANDARD;
+        TranscriptionTypeEnum transcriptionTypeEnum = TranscriptionTypeEnum.COURT_LOG;
+
+        TranscriptionRequestDetails transcriptionRequestDetails = createTranscriptionRequestDetails(
+            hearing.getId(), courtCase.getId(), transcriptionUrgencyEnum.getTranscriptionUrgencyKey(),
+            transcriptionTypeEnum.getTranscriptionTypeKey(), TEST_COMMENT, START_TIME, END_TIME
+        );
+
+        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT_URI)
+            .header("Content-Type", "application/json")
+            .content(objectMapper.writeValueAsString(transcriptionRequestDetails));
+
+        mockMvc.perform(requestBuilder).andExpect(status().isOk());
+
+    }
+
+    @Test
+    @Order(2)
+    @Transactional
+    void transcriptionRequestWithNullDatesAndSentencingRemarksTypeShouldReturnSuccess()
+        throws Exception {
+        TranscriptionUrgencyEnum transcriptionUrgencyEnum = TranscriptionUrgencyEnum.STANDARD;
+        TranscriptionTypeEnum transcriptionTypeEnum = TranscriptionTypeEnum.SENTENCING_REMARKS;
+
+        TranscriptionRequestDetails transcriptionRequestDetails = createTranscriptionRequestDetails(
+            hearing.getId(), courtCase.getId(), transcriptionUrgencyEnum.getTranscriptionUrgencyKey(),
+            transcriptionTypeEnum.getTranscriptionTypeKey(), TEST_COMMENT, null, null
+        );
+
+        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT_URI)
+            .header("Content-Type", "application/json")
+            .content(objectMapper.writeValueAsString(transcriptionRequestDetails));
+
+        mockMvc.perform(requestBuilder).andExpect(status().isOk());
+
+    }
+
+    @Test
+    @Order(3)
+    @Transactional
+    void transcriptionRequestWithNullHearingAndNullCaseShouldThrowException() throws Exception {
+        TranscriptionUrgencyEnum transcriptionUrgencyEnum = TranscriptionUrgencyEnum.STANDARD;
+        TranscriptionTypeEnum transcriptionTypeEnum = TranscriptionTypeEnum.COURT_LOG;
+
+        TranscriptionRequestDetails transcriptionRequestDetails = createTranscriptionRequestDetails(
+            null, null, transcriptionUrgencyEnum.getTranscriptionUrgencyKey(),
+            transcriptionTypeEnum.getTranscriptionTypeKey(), TEST_COMMENT, START_TIME, END_TIME
+        );
+
+        MockHttpServletRequestBuilder requestBuilder = post(ENDPOINT_URI)
+            .header("Content-Type", "application/json")
+            .content(objectMapper.writeValueAsString(transcriptionRequestDetails));
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(header().string("Content-Type", "application/problem+json"))
+            .andExpect(status().isInternalServerError())
```

Comment:
This doesn't feel right to me - think it should be a client error code - can we have 400 Bad Request perhaps?